app/controllers/admin/config/fonts_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+class Admin::Config::FontsController < Admin::AdminController
```

Comment:
Before we had branding controller because there was idea of having one save button for logos and fonts. Later we decided to switch into 2 sections, so fonts and logos can be moved to separate controllers to keep it clean.